### PR TITLE
Habib/34905/blog homepage

### PR DESCRIPTION
**Pull request by ShuvoHabib**
Changes:

-  Blog Homepage Carousel Banner (Hero)

## Type of change

-   [ ] Bug fix
-   [x] New feature
-   [ ] Update feature
-   [ ] Refactor code
-   [ ] Translation to code
-   [ ] Translation to crowdin
-   [ ] Script configuration
-   [ ] Improve performance
-   [ ] Style only
-   [ ] Dependency update
-   [ ] Documentation update
-   [ ] Release

### CODE REVIEW

**Review by arome-binary — COMMENTED** [review]
LGTM

**Review by arome-binary — APPROVED** [review]

File: src/pages/blog/components/_hero.js [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+    width: 100%;
+    height: 100%;
+    object-fit: cover;
+
+    @media ${device.tabletL} {
+        display: flex;
+    }
+`
+const Wrapper = styled(Flex)`
+    position: relative;
```
Comment (njazuli-deriv) [inline src/pages/blog/components/_hero.js:42]:
```suggestion
    position: relative;
    max-width: 1440px;
    margin: 0 auto;
```

Comment (njazuli-deriv) [reply src/pages/blog/components/_hero.js:42]: Maybe we can set the Wrapper's max-width to 1440px as it stretched to the left side on bigger screen (>1600)

image

Comment (sean-binary) [reply src/pages/blog/components/_hero.js:42]: Maybe let's keep it to 1600px to follow that of the header and footer.

Comment (ShuvoHabib) [reply src/pages/blog/components/_hero.js:42]: Hi @sean-binary @njazuli-deriv Discussed with Dennis, he want's this full width

Comment (sean-binary) [reply src/pages/blog/components/_hero.js:42]: Nono, we meant the text container. Not the background image.

File: src/pages/blog/components/_hero.js [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+        padding: 2rem;
+    }
+    @media ${device.tabletL} {
+        height: ${(props) => props.tabletL_height};
+        align-items: flex-start;
+    }
+    @media ${device.tablet} {
+        justify-content: center;
+        height: 100%;
+    }
```
Comment (sean-binary) [inline src/pages/blog/components/_hero.js:82]: Seems like we're not using this.

File: src/pages/blog/components/_hero.js [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+    max-width: 1440px;
+    margin: 0 auto;
+
+    h1 {
+        margin-bottom: 20px;
+    }
+    @media screen and (min-width: 1600px) {
+        max-width: 1440px;
+        margin: 0 auto;
+    }
```
Comment (sean-binary) [inline src/pages/blog/components/_hero.js:62]: Don't think we need this.

File: src/pages/blog/components/_hero.js [hunk 4]
```diff
@@ -0,0 +1,120 @@
...
+    @media screen and (min-width: 1900px) {
+        max-width: 1900px;
+        margin: 0 auto;
+    }
```
Comment (sean-binary) [inline src/pages/blog/components/_hero.js:66]:
```suggestion
    @media screen and (min-width: 1980px) {
        max-width: 1900px;
    }
```

File: src/pages/blog/components/_hero.js [hunk 5]
```diff
@@ -0,0 +1,99 @@
...
+        position: relative;
+        z-index: 99;
+    }
+
+    @media ${device.tablet} {
+        height: 400px;
+        text-align: left;
+        object-fit: cover;
+    }
+`
```
Comment (sean-binary) [inline src/pages/blog/components/_hero.js:28]: I don't think we're using this as well 🙏  Rest LGTM

Comment (ShuvoHabib) [reply src/pages/blog/components/_hero.js]: Thanks for the review. Only the responsive part is used

**Review by sean-binary — APPROVED** [review]

**Pull request merged**